scalameta/contrib/src/test/scala/scala/meta/contrib/ScaladocParserSuite.scala

```diff
@@ -51,6 +51,31 @@ class ScaladocParserSuite extends FunSuite {
     assert(
       parseString(
         s"""
+         // $expectedBody
```

Comment:
Also these tests should have their own section. they have nothing to do with indentation